src/app/modules/components/courses/course-list/course-list.component.ts

```diff
@@ -10,30 +11,33 @@ import { CoursesService } from '@core/services/courses.service';
 })
 export class CourseListComponent implements OnInit {
 
-  public courses: Course[] = [];
+  public courses: Observable<any[]>;
```

Comment:
Do not use any. And add $ to the observable name: public courses$: Observable<Course[]>;